apps/backend/app/services/feature_store_service.py

```
@@ -0,0 +1,715 @@
+        elif dataset.file_type in ["xls", "xlsx"]:
+            file_io = io.BytesIO(file_bytes)
+            df = pd.read_excel(file_io)
+        else:
+            raise ValidationError(
+                message=f"Unsupported file type: {dataset.file_type}",
+                details={"file_type": dataset.file_type}
+            )
+
+        # Apply feature using FeatureEngineer
+        engineer = FeatureEngineer()
+        result_df = await engineer.apply_stored_feature(df, feature)
```

Ruff (0.14.10)

Local variable `result_df` is assigned to but never used

Remove assignment to unused variable `result_df`

(F841)